pkg/virt-handler/vm.go

```diff
@@ -547,6 +557,42 @@ func (c *VirtualMachineController) Execute() bool {
 	return true
 }
 
+func (d *VirtualMachineController) getMigrationConfig(vmi *v1.VirtualMachineInstance) (migrationConfig *v1.MigrationConfig) {
+	progressTimeout := int64(150)
+	completionTimeoutPerGiB := int64(800)
+	migrationConfig = &v1.MigrationConfig{
+		CompletionTimeoutPerGiB: completionTimeoutPerGiB,
+		ProgressTimeout:         progressTimeout,
+	}
+
+	migrationObj := d.getVMIMigrationFromCache(vmi)
+	if migrationObj != nil {
+
+		conf := migrationObj.Spec.Config
+
+		if conf != nil {
+			if conf.CompletionTimeoutPerGiB != 0 {
+				migrationConfig.CompletionTimeoutPerGiB = conf.CompletionTimeoutPerGiB
+			}
+			if conf.ProgressTimeout != 0 {
+				migrationConfig.ProgressTimeout = conf.ProgressTimeout
+			}
+		}
+	}
+	return
+}
+
+func (d *VirtualMachineController) getVMIMigrationFromCache(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstanceMigration {
+	objs := d.vmiMigrationInformer.GetStore().List()
```

Comment:
@davidvossel Is it better to use an informer or is it better to get rid of this and simply to query with a client, like: 

```suggestion
	migrations := d.clientset.VirtualMachineInstanceMigration(vmi.Namespace)
	objs, err := migrations.List(&v12.ListOptions{})
```
I thought I'll need the informer in #2059 , but I that I don't need it there.